lib/browser/init.ts

```diff
@@ -4,6 +4,9 @@ import * as fs from 'fs';
 import { Socket } from 'net';
 import * as path from 'path';
 import * as util from 'util';
+
+// Emit 'exit' event on quit.
+import { app } from 'electron/main';
```

Comment:
This can't work? The `electron` import isn't available until `reset-search-paths` and `common/init` are called